src/shared/lib/geography/index.js

```diff
@@ -104,7 +104,8 @@ export const isCity = function(location) {
 };
 
 export const getSmallestLocationStr = function(location) {
-  const smallestStr = location.city || location.county || location.state || location.country;
+  const county = !location.countyId ? location.county : false;
```

Comment:
I guess that's the aggregate case, right? Why don't we overwrite the county string, like in other places?

Reply:
Yes, this is the `TriCounties, Utah` case. It was causing issues to leave the IDs in `county`, since the name cannot be resolved, and we already have made a human readable name